frontend/app/src/store/blockchain/index.ts

```diff
@@ -219,7 +219,7 @@ export const useBlockchainStore = defineStore('blockchain', () => {
 
   const { getAssetAssociationIdentifiers, getAssociatedAssetIdentifier } = useAssetInfoRetrieval();
 
-  const getBreakdown = (asset: string, chains?: string[], groupId?: string): AssetBreakdown[] => {
+  const getBreakdown = (asset: string, isLiability = false, chains?: string[], groupId?: string): AssetBreakdown[] => {
```

Comment:
In this part it probably makes sense to keep it for reusability but maybe instead of exporting this just keep it internal and export `getAssetBreakdown` `getLiabilityBreakdown` that hide this from the consumer.